openprescribing/frontend/management/commands/measure_definitions/bens.json

```diff
@@ -273,10 +270,11 @@
             "as a percentage of prescribing of all statins"
         ],
         "description": [
-            "TBA"
+            "Prescribing of high-cost statins ",
+            "as a percentage of prescribing of all statins."
         ],
         "why_it_matters": [
-            "TBA"
+            "It is recommended that when treating high cholesterol doctors prescribe statins that both reduce cholesterol by at least 40%, and that are low-cost.  The low-cost statins (simvastatin and atorvastatin) are suitable for the majority of patients."
```

Comment:
Hard to parse. Perhaps:

"When treating high cholesterol, doctors are recommended to prescribe statins which are both low-cost, and reduce cholesterol by at least 40%" ?